test/test_interactive.py

```diff
@@ -311,33 +311,55 @@ def test_audio_worklet_stereo_io(self):
     os.mkdir('audio_files')
     shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
     shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
-    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-DTEST_AND_EXIT=1'])
```

Comment:
With interactive tests so you want `TEST_AND_EXIT` defined?  I thought maybe the point of that macro was for use in the browser tests, not the interactive tests.

If its defiend in both then maybe we don't need it at all?

Reply:
If interactive is meant to run continuously then it can be removed. I wasn't sure of the use case.

Reply:
I guess interactive needs the user to shut it down, in which case I'll remove it here and delete the one or 10 second selection.